index.d.ts

```diff
@@ -125,7 +125,7 @@ export type DailyNonFatalErrorType =
 export type DailyNetworkTopology = 'sfu' | 'peer';
 
 export interface DailyParticipantsObject {
-  local: DailyParticipant;
+  local?: DailyParticipant;
```

Comment:
i'll let @kompfner weigh in here, but I think maybe the more accurate thing to do would be to change the return of `participants()`(line 1105) to return `DailyParticipantsObject || {}`.  The only time `local` can be undefined is before `preAuth`/`startCamera`/`join` or after `leave` (basically.. not in a call) and in this case, `participants()` will return `{}`. The other uses of `DailyParticipantsObject` come at times when you are in a call and should be guaranteed to have a `local` participant.  With the change I'm proposing, we can keep this guarantee.